pilight/pilight.py

```diff
@@ -103,6 +105,11 @@ def __init__(self, host='127.0.0.1', port=5000, timeout=1,
                 answer_1, answer_2)
 
         self.callback = None
+        self.protocol_registry = self._load_protocol_schema()
+
+    def _load_protocol_schema(self):
```

Comment:
Since this is a essential function to be used externally I would remove the _ prefix.